### ISSUE TITLE
Consider using true/false for all booleans in docs

### ISSUE THREAD

**Issue by kristianheljas** [CiscoDevNet/ansible-meraki#443]
Based on the [community decision](https://github.com/ansible-community/community-topics/discussions/120) to use `true/false` for boolean values in documentation and examples, we ask that you evaluate booleans in this collection and consider changing any that do not use `true/false` (lowercase).

See [documentation block format](https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#documentation-block) for more info (specifically, option defaults).

If you have already implemented this or decide not to, feel free to close this issue.

---

P.S. This is auto-generated issue, please raise any concerns [here](https://github.com/ansible-community/community-team/issues/60)

**Comment by kbreit** [comment]
I will migrate to `true/false` instead of `yes/no` but it will take some time. This issue will be kept open until they're all resolved.

**Comment by y0rune** [comment]
Yeah, in the that weekend I will have some time so I am going to finish the task with the warnings!
In the #361, we have changes from `yes/no` to `true/false`. But you have right to still open that issue until we resolve it.
